### PR TITLE
updated

### PR DESCRIPTION
**Pull request by sachinsharma263**
Hy,

 please review code.
i have added pojo class.

### CODE REVIEW

File: src/main/java/com/qa/trcrm/pages/ContactsPage.java [hunk 1]
```diff
@@ -44,5 +45,18 @@ public String addPerson(String firstName,String emailId) {
 		
 		return util.doGetText(personAddedMsg);
 	}
+	public String addPerson(Contacts contacts)
+	{
+		util.waitForPresenceOfElementLocated(addPersonButton);
+		util.doClick(addPersonButton);
+util.waitForPresenceOfElementLocated(name);
```
Comment (sachinautomation25) [inline src/main/java/com/qa/trcrm/pages/ContactsPage.java:52]: Please do proper formating

**Review by sachinautomation25 — COMMENTED** [review]
please chk reviews

File: src/main/java/com/qa/trcrm/pages/ContactsPage.java (hunk 1 above)
Comment (sachinsharma263) [reply src/main/java/com/qa/trcrm/pages/ContactsPage.java:52]: done

**Comment by sachinautomation25** [comment]
All seems good, merging done.

**Pull request merged**